WordPress/lint.xml

```diff
@@ -24,9 +23,6 @@
     <issue id="IconDipSize" severity="error" />
     <issue id="InconsistentArrays" severity="error" />
     <issue id="StringFormatCount" severity="error" />
-
-    <issue id="TypographyEllipsis" severity="error"/>
-
```

Comment:
Removing this because it seems to be a duplicate and is already present on line 55/52 below, so keeping only one.